src/parsing.jl

```diff
@@ -106,9 +106,9 @@ function get_source_fun(function_expr)
     # recursive step for begin :a + :b end
     if function_expr isa Expr &&
         function_expr.head == :block &&
-        length(function_expr.args) == 2 # omitting the line number node
```

Comment:
I just call `Base.remove_linenums!` wherever possible, which changed the way we index expressions in a few places.